backend/src/app/modules/course/course.service.ts

```
@@ -89,7 +90,11 @@
     if (process.env.NODE_ENV === 'development') {
         delete whereTerms.isPublished
     }
-    let sortBy = [] as any
+    let sortBy = [
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'sortBy' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)